### ISSUE TITLE
"def finalize(data):" Function implementation mistake

### ISSUE THREAD

**Issue by wingwing518** [Steven-Hewitt/QA-with-Tensorflow#1]
Hi Steven,

I found a mistake in your code, which locate in the "In [9]" on  "QA with Tensorflow.ipynb".
In the function definition of "def finalize(data):",
the line `for cqas in train_data:` should be replaced with `for cqas in data:`. Otherwise, after the finalize function, all the training set and test set will be referring to the same training data.

And later on, you can obtain a >90% accuracy is just because you are testing the model with your training set, although the name of the variable is "test set". 

Throughout my experiment, I found that your correct setting should obtain around 60% accuracy in the test set.

Anyways, it is a good tutorial and I like it.

Thanks a lot.

Regards,
Wing

**Comment by JustinHallquist** [comment]
Just want to verify the bug as well as the drastic drop in expected accuracy (~65% over 300000 iterations) once corrected.

I highly recommend updating the documentation with the corrected code and updated accuracy expectations. The framework currently provided will need adjustments to hit the projected 90+% mentioned.

Great tutorial, and thank you so much for preparing this

**Comment by Steven-Hewitt** [comment]
Oh hey, somehow I missed this back when it was first posted.  Unfortunately, I'm no longer updating this tutorial, but if I get a pull request to fix this I'll accept it.

**Comment by JustinHallquist** [comment]
fixed with https://github.com/Steven-Hewitt/QA-with-Tensorflow/pull/2
feel free to close this
